.env.example

```
@@ -86,3 +86,9 @@ CALLBACK_READ_TIMEOUT = 10
 
 # require as a env if you want to use doc transformation
 OPENAI_API_KEY=""
+
+# Resource Monitoring Configuration
+RESOURCE_MONITORING_ENABLED=True
+RESOURCE_CHECK_INTERVAL=5
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RESOURCE_CHECK_INTERVAL key should go before the RESOURCE_MONITORING_ENABLED key

(UnorderedKey)